stubs/nanoleafapi/nanoleafapi/nanoleaf.pyi

```diff
@@ -0,0 +1,68 @@
+from _typeshed import Incomplete
+from collections.abc import Callable
+from typing import Any
+
+RED: tuple[int, int, int]
+ORANGE: tuple[int, int, int]
+YELLOW: tuple[int, int, int]
+GREEN: tuple[int, int, int]
+LIGHT_BLUE: tuple[int, int, int]
+BLUE: tuple[int, int, int]
+PINK: tuple[int, int, int]
+PURPLE: tuple[int, int, int]
+WHITE: tuple[int, int, int]
+
+class Nanoleaf:
+    ip: str
+    print_errors: bool
+    url: str
+    auth_token: str | None
```

Comment:
I don't think it's possible to create a `Nanoleaf` instance where `auth_token` is `None`, as it looks like an exception is raised in `__init__` if `auth_token` is `None`:

https://github.com/MylesMor/nanoleafapi/blob/145e0413606cb3f34337e7c36cfb67996a7ef9cb/nanoleafapi/nanoleaf.py#L51-L56

```suggestion
    auth_token: str
```